lib/scan/options.rb

```diff
@@ -73,6 +73,16 @@ def self.available_options
                                      env_name: "SCAN_BUILDLOG_PATH",
                                      description: "The directory were to store the raw log",
                                      default_value: "~/Library/Logs/scan"),
+        FastlaneCore::ConfigItem.new(key: :derived_data_path,
+                                     short_option: "-e",
```

Comment:
Could you change away from `-e`, I just recently added this short option to another option